kotlin/android/app/src/main/java/dev/firezone/android/features/customuri/ui/CustomUriHandlerActivity.kt

```diff
@@ -41,4 +49,10 @@ class CustomUriHandlerActivity : AppCompatActivity(R.layout.activity_custom_uri_
             finish()
         }
     }
+
+    private fun notifyError(message: String) {
+        val notification = CustomUriNotification.update(this, CustomUriNotification.Error(message)).build()
+        val manager = getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        manager.notify(CustomUriNotification.ID, notification)
```

Comment:
weird to get the notification manager here, but also in the `CustomUriNotification` object. This function is doing both High Level and Low Level (using CustomUriNotification to build the notification, but also not using it to "notify")

Reply:
Also some repeating myself with the `TunnelStatusNotification`. Not sure of a good way to consolidate them both yet :/